oldsite/static/js/toggle.js

```diff
@@ -0,0 +1,15 @@
+const currentPagePath = window.location.pathname;
+const currentPageName = currentPagePath.substring(currentPagePath.lastIndexOf('/') + 1);
+
+const pageToggle = document.getElementById('page-toggle');
+
+if (pageToggle.hasAttribute('data-target')) {
+  const redirectPageName = pageToggle.getAttribute('data-target');
+  pageToggle.addEventListener('click', function() {
+    window.location.href = '../' + redirectPageName;
+  });
+} else {
+  pageToggle.addEventListener('click', function() {
+    window.location.href = '../' + currentPageName;
+  });
+};
```

Comment:
How about adding a global `if` whether `pageToggle` was found, and simplifying the `if`?
```suggestion
if (pageToggle) {
  const redirectPageName = pageToggle.hasAttribute('data-target') ? pageToggle.getAttribute('data-target') : currentPageName;
  pageToggle.addEventListener('click', function() {
    window.location.href = '../' + redirectPageName;
  });
}
```